### PR TITLE
 Add support for amd operator installation

### PR DESCRIPTION
**Pull request by tarukumar**

### CODE REVIEW

File: ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh [hunk 1]
```diff
@@ -12,10 +12,10 @@ CSVNAME="$(oc get packagemanifests/gpu-operator-certified -n openshift-marketpla
 sed -i -e "0,/v1.11/s//$CHANNEL/g" -e "s/gpu-operator-certified.v1.11.0/$CSVNAME/g"  "$GPU_INSTALL_DIR/gpu_install.yaml"
 
 oc apply -f "$GPU_INSTALL_DIR/gpu_install.yaml"
-
+oc apply -f "$GPU_INSTALL_DIR/nfd_operator.yaml"
```
Comment (bdattoma) [inline ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh:15]: for NVIDIA GPUs, NFD operator gets deployed by NVIDIA operator right?

File: ods_ci/tasks/Resources/Provisioning/GPU/amd_operator/kmm_operator_install.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+metadata:
+  name: openshift-kmm
+
+---
+
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-kmm-operator-group
+  namespace: openshift-kmm
```
Comment (bdattoma) [inline ods_ci/tasks/Resources/Provisioning/GPU/amd_operator/kmm_operator_install.yaml:12]: isn't "openshift" a protected name for namespaces? meaning we cannot create namespaces starting with "openshift"? maybe I misremember

File: ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh (hunk 1 above)
Comment (tarukumar) [reply ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh:15]: The NFD operator  is an independent operator and is not installed  by the Nvidia operator.

File: ods_ci/tasks/Resources/Provisioning/GPU/amd_operator/kmm_operator_install.yaml (hunk 2 above)
Comment (tarukumar) [reply ods_ci/tasks/Resources/Provisioning/GPU/amd_operator/kmm_operator_install.yaml:12]: I don't think so

Comment (bdattoma) [reply ods_ci/tasks/Resources/Provisioning/GPU/amd_operator/kmm_operator_install.yaml:12]: checked, the limitation is only on OpenShift Projects, not namespaces

**Review by bdattoma — APPROVED** [review]

**Review by mwaykole — APPROVED** [review]

**Pull request merged**